src/Elastic.Apm/Model/Span.cs

```diff
@@ -406,5 +408,23 @@ private Destination DeduceHttpDestination()
 				return null;
 			}
 		}
+
+		public void SetLabel(string key, string value)
+			=> Context.Labels.Add(key, value);
```

Comment:
should this be assignative i.e. use the indexer?

Reply:
Right, fixed this - and also added a test which covers this.